start.py

```diff
@@ -68,17 +70,29 @@ def main(args):
                         # Secure cluster needs the ticket to execute rest of commands
                         # after cluster start.
                         environment=['MAPR_TICKETFILE_LOCATION=/opt/mapr/conf/mapruserticket']
-                        if args.secure else [])
+                        if args.secure or args.kerberos else [])
 
     secondary_nodes = [Node(hostname=hostname,
                             group='secondary',
                             image=secondary_node_image,
                             devices=node_disks.get(hostname))
                        for hostname in args.secondary_nodes]
 
-    cluster = Cluster(primary_node, *secondary_nodes)
+    nodes = [primary_node] + secondary_nodes
+    if args.kerberos:
+        logger.info('Creating KDC node...')
+        kdc = kerberos_helper.Kerberos_Helper(args.network, args.operating_system)
```

Comment:
Any reason not to call the class Kdc in the first place if it'll be instantiated and treated like one?

Reply:
Changed the instance variable name.

Reply:
> Just tiny suggestions. Can we update the README to give an example of using the new args?

Done.